src/Swarm/Game/CESK.hs

```diff
@@ -373,3 +378,8 @@ instance Show RobotUpdate where show _ = "RobotUpdate {???}"
 instance Eq WorldUpdate where _ == _ = True
 
 instance Eq RobotUpdate where _ == _ = True
+
+instance FromJSON WorldUpdate where parseJSON _ = pure $ WorldUpdate $ \w -> Right w
```

Comment:
I guess this needs to be updated with the actual update function after decoding.

Reply:
I think `WorldUpdate` is my mistake. It is too generic when all we use it for is updating one robot or cell.

We should add more concrete constructors that would be possible to serialize and also easier to debug.

But it does not have to be in this PR.

Reply:
Ok, I've added a TODO